src/lib/list/_list-theme.scss

```diff
@@ -8,14 +8,22 @@
   $background: map-get($theme, background);
   $foreground: map-get($theme, foreground);
 
-  .mat-list, .mat-nav-list {
+  .mat-list, .mat-nav-list, .mat-selection-list {
     .mat-list-item {
       color: mat-color($foreground, text);
     }
 
+    .mat-list-option {
+      color: mat-color($foreground, text);
+    }
+
     .mat-subheader {
       color: mat-color($foreground, secondary-text);
     }
+
+    .mat-list-item-disabled {
+      background: #ededed;
```

Comment:
Use `mat-color` for disabled background? Like in https://github.com/angular/material2/blob/0409707adc3dd3114730310a95e4f9ac1827b3dd/src/lib/button-toggle/_button-toggle-theme.scss#L32